src/solvers/dgsem_tree/indicators.jl

```diff
@@ -226,6 +226,52 @@ function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorMax)
   end
 end
 
+"""
+    IndicatorClamp(equations::AbstractEquations, basis; a=0.0, b=1.0, variable)
+    IndicatorClamp(semi::AbstractSemidiscretization; a=0.0, b=1.0, variable)
```

Comment:
I am wondering whether `min` and `max` instead of `a` and `b` would clarify the intent.

Reply:
Agree. Changed accordingly.